src/data/projects/newrelic-video-agent-roku.json

```
@@ -10,7 +10,7 @@
   "supportUrl": "https://discuss.newrelic.com/t/new-relic-open-source-roku-agent/97802",
   "githubUrl": "https://github.com/newrelic/video-agent-roku",
   "permalink": "https://opensource.newrelic.com/projects/newrelic/video-agent-roku",
-  "iconUrl": null,
+  "iconUrl": "https://github.com/newrelic/video-agent-rokun/blob/master/icon.png?raw=true",
```

Hey @adwetzelnr 👋 Looks like a typo here:
```suggestion
  "iconUrl": "https://github.com/newrelic/video-agent-roku/blob/master/icon.png?raw=true",
```

---

Thanks for the heads up.  I noticed that too and submitted another PR.  Is there a better way to address/amend a PR that has a problem?